examples/stable/couchbase/couchbase-blueprint.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: couchbase-blueprint
+actions:
+  backup:
+    kind: CustomResource
+    outputArtifacts:
+      cbBackup:
+        keyValue:
+          path: '/couchbase-backups/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/backup.tgz'
+    phases:
+    - func: KubeTask
+      name: TakeBackup
+      objects:
+        authSecret:
+          kind: Secret
+          name: "{{ .Object.spec.authSecret }}"
+          namespace: "{{ .Object.metadata.namespace }}"
+      args:
+        namespace: "{{ .Object.metadata.namespace }}"
+        image: kanisterio/couchbase-tools:0.22.0
+        command:
+          - bash
+          - -o
+          - pipefail
+          - -o
+          - errexit
+          - -c
+          - |
+            rm -rf /backups/test
+            mkdir -p /backups/test
+            cbbackupmgr config --archive /backups/test --repo couchbase
+            cbbackupmgr backup                                              \
+              --archive /backups/test                                       \
+              --repo couchbase                                              \
+              --cluster couchbase://{{ .Object.metadata.name }}.{{ .Object.metadata.namespace }}.svc    \
+              --username {{ .Phases.TakeBackup.Secrets.authSecret.Data.username | toString }} \
+              --password {{ .Phases.TakeBackup.Secrets.authSecret.Data.password | toString }}
+            tar zcvf - -C /backups/ test | kando location push --profile '{{ toJson .Profile }}' --path '/couchbase-backups/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}/backup.tgz' -
```

Comment:
@PrasadG193 
is the space between `/backups/` and `test` is expected for the command
```
tar zcvf - -C /backups/ test 
```